apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/commons/RetryOnFailoverOrConnectionExceptionIT.java

```diff
@@ -75,7 +76,7 @@ static class RetryableOperationBean {
      * @throws Exception whichever {@link Exception} that is at the top of the provided {@link
      *     Deque} of {@link Exception}s, if not empty
      */
-    @RetryOnRDSFailover(backoff = @Backoff(delay = 1))
+    @RetryOnFailoverOrConnectionException(backoff = @Backoff(delay = 1), maxAttempts = 3)
```

Comment:
Is there any value in having "3" defined as a relevant constant or configuration parameter in a common "area" so that the value may be leveraged consistently if a change is needed?

Reply:
Hmm... I personally feel it should be OK to keep the `maxAttempts` at 5. I haven't seen any indication in my testing that it needs to be changed. Also, since this is defined as an annotation I don't know of a good way to inject a value here from configuration. AFAIK Java annotation values need to be constants.